### PR TITLE
Add a test script to the docs

### PR DESCRIPTION
**Pull request by chrissvo**
Fixes: https://github.com/mgmco/bugs/issues/2361

Changes proposed in this pull request:
 * TSIA

More readable:
screen shot 2017-02-09 at 12 05 19

cc @mgmco/developers

### CODE REVIEW

File: _testing/test_script.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Test script
+position: 2
+---
+This is a simple test script that will allow you to check whether you integrated successfully
+
+| Step | Objective                                    | Action | Expected Result |
+| ---- | -------------------------------------------- | ------ | --------------- |
+| 1    | Check DNS change                             | 1. Open terminal<br>2. Execute <code>$ dig{% raw %}{{subdomain}}.{{clientdomain}}{% endraw %}.nl CNAME</code> | The answer section should contain:<br><code>;; ANSWER SECTION:<br>{% raw %}{{subdomain}}.{{clientdomain}}.nl. {{port}} IN  CNAME {{subdomain}}.membergetmember.co.{% endraw %}</code> |
```
Comment (harianus) [inline _testing/test_script.md]: Missing a space

```
$ dig{% raw %}
     ^
```

And maybe just leave out the `$`?

Comment (chrissvo) [inline _testing/test_script.md]: You always see that `$` online though right?

Comment (mtimofiiv) [inline _testing/test_script.md]: I kind of agree with @chrissvo here, it makes it obvious to people that know command line that this is a command line thing.

Comment (harianus) [inline _testing/test_script.md]: K'den

**Pull request merged**